updater/osv/osv.go

```diff
@@ -570,6 +571,7 @@ func (e *ecs) Insert(ctx context.Context, skipped *stats, name string, a *adviso
 						if err == nil {
 							nv := ver.IncPatch()
 							v.Range.Upper = claircore.FromSemver(&nv)
+							v.FixedInVersion = nv.Original()
```

Comment:
I'd be ok with leaving this out. We don't actually know the name of the version in which this is fixed